usaspending_api/recipient/v2/views/recipients.py

```diff
@@ -209,16 +211,19 @@ def extract_business_categories(recipient_name, recipient_duns):
         business_categories |= set(get_business_categories(business_types, data_type="fpds"))
 
     # combine with latest transaction's business categories
-    le_business_cat = (
-        LegalEntity.objects.filter(recipient_name=recipient_name, recipient_unique_id=recipient_duns)
-        .order_by("-update_date")
+    # from usaspending_api.references.models import LegalEntity
+    latest_transaction = (
+        UniversalTransactionView.objects.filter(
+            recipient_hash=recipient_hash, action_date__gte=settings.API_SEARCH_MIN_DATE
+        )
+        .order_by("-action_date", "-transaction_id")
```

Comment:
possibly out-of-scope, but would it be practical to put this logic in a function somewhere, so that instead of always needing to remember this two part sort, a dev could just use 
```
latest_transaction = latest_transaction(recipient_hash)
```
?

Reply:
It's possible but not practical.  I don't know how the consumer wants to sort things.

Reply:
If I'm correct, this is the logic for lastest_transaction on an award. That's not a user supplied sort; it's internal business logic

Reply:
This is not the logic for latest transaction.

Reply:
ohh